gulpfile.js

```diff
@@ -33,6 +33,7 @@ gulp.task('test', ['set-test-env'], function(cb) {
 
     gulp.src('test/**/*.js')
         .pipe(plumber())
+        .pipe(istanbul())
```

Comment:
What is this doing, and why is it needed?

Reply:
Oh, I forgot to remove it, I was reading the gulp-istanbul documentation and I tested it and I forgot to remove it, sorry :)

Reply:
No worries, I'll take care of it.